docs/spec/api.md

```diff
@@ -2055,7 +2055,7 @@ The following headers will be returned with the response:
 
 |Name|Description|
 |----|-----------|
-|`Range`|Range indicating the current progress of the upload.|
+|`Range`|Range indicating the current progress of the upload. In 2.0.x offset is always zero since resumable push is not supported.|
```

Comment:
We should have a separate document that enumerates the current implementation status.